code/processes/tailreader.q

```diff
@@ -0,0 +1,33 @@
+\d .tr 
+partitiontype:@[value;`partitiontype;`date];                               /-set type of partition (defaults to `date)
+gmttime:@[value;`gmttime;1b];                                              /-define whether the process is on gmttime or not
+getpartition:@[value;`getpartition;
+  {{@[value;`.wdb.currentpartition;
+    (`date^partitiontype)$(.z.D,.z.d)gmttime]}}];                      /-function to determine the partition value
+segmentid: "J"$.proc.params[`segid]
```

Comment:
Is this a string in .proc.params?

Reply:
Yes, seems all values in the .proc.params dict are strings, except the `debug value which was an empty list.